http/src/main.rs

```diff
@@ -73,20 +73,14 @@ fn main() {
                 std::process::exit(1);
             }
 
-            let result = config::initialize(&home, bits, profile);
+            let result = config::init(&home, bits, profile);
 
             match result {
-                Ok(_) => {
-                    let (kp, _, _) = std::fs::File::open(config_path)
```

Comment:
What's the rationale for *not* reopening?

If I remember correctly I added this as sanity check for the roundtrip working. This would be the `LoadingError::PeerIdMismatch`.

Reply:
After re-reading this a few times it seems that I must have only used some println logging for confirming the peerid before writing and after re-opening, ok to change this as it never asserted anything else than written file can be loaded.